exampleapp/src/main/res/values/colors.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <color name="primary">#F44336</color>
```

Comment:
Sure, we should do that after I implement this functionality inside the SDK. These colors are just standard material colors from Googles material palette.